tests/cases/fourslash/refactorConvertParamsToDestructuredObject_interfaceNoIntersection.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path='fourslash.ts' />
+
+////interface IFoo {
+////    method(x: string, y: string): void;
+////}
+////interface IBar {
+////    method(x: number): void;
+////}
+////const x: IFoo & IBar = {

```

Comment:
what happens with unions? Do we even get the error that triggers the codefix?

Reply:
There is a test refactorConvertParamsToDestructuredObject_interfaceNoUnion.ts:
https://github.com/microsoft/TypeScript/pull/42652/files#diff-bb37f20a7f9bacc80fda3468d1e6c808314c79f47e8f58dc3d55c7e3d5447360

It's a refactor so we just don't offer it.